infrastructure/sit_template.py

```diff
@@ -75,8 +80,15 @@ def init_template(self):
                             "ecs:DeregisterContainerInstance",
                             "ecs:DiscoverPollEndpoint",
                             "ecs:Submit*",
-                            "ecs:Poll"
-                        ],  
+                            "ecs:Poll",
+                            "ecs:StartTelemetrySession",
+                            "ecr:GetAuthorizationToken",
+                            "ecr:BatchCheckLayerAvailability",
+                            "ecr:GetDownloadUrlForLayer",
+                            "ecr:BatchGetImage",
+                            "logs:CreateLogStream",
+                            "logs:PutLogEvents"
+                        ],
```

Comment:
These roles are required for ECS Instances to report stats to CloudWatch. Every Node in the ECS Cluster is an EC2 instance which runs a Container Agent that communicates with AWS using this Role/Policy. 
See http://docs.aws.amazon.com/AmazonECS/latest/developerguide/cloudwatch-metrics.html#enable_cloudwatch